overlord/configstate/configcore/services.go

```diff
@@ -372,8 +372,20 @@ func handleServiceConfigSSHListen(dev sysconfig.Device, tr ConfGetter, opts *fsO
 
 	if opts == nil {
 		sysd := systemd.New(systemd.SystemMode, &sysdLogger{})
-		if err := sysd.ReloadOrRestart([]string{"ssh.service"}); err != nil {
-			return err
+		// From 22.10 sshd now uses socket based activation. This changes how to reload
+		// ssh configuration
+		// Discussion here: https://discourse.ubuntu.com/t/sshd-now-uses-socket-based-activation-ubuntu-22-10-and-later/30189/9
+		if enabled, err := sysd.IsEnabled("ssh.socket"); err == nil && enabled {
+			if err := sysd.DaemonReload(); err != nil {
```

Comment:
Presume this is not applicable anymore: https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1819728

Reply:
Nope, atleast not from observations, they seem to indicate things are fine